core/web/solana_transfer_controller.go

```diff
@@ -0,0 +1,140 @@
+	App chainlink.Application
+}
+
+// Create sends Luna and other native coins from the Chainlink's account to a specified address.
+func (tc *SolanaTransfersController) Create(c *gin.Context) {
+	solanaChains := tc.App.GetChains().Solana
+	if solanaChains == nil {
+		jsonAPIError(c, http.StatusBadRequest, ErrSolanaNotEnabled)
+		return
+	}
+
+	var tr solanamodels.SendRequest
+	if err := c.ShouldBindJSON(&tr); err != nil {
+		jsonAPIError(c, http.StatusBadRequest, err)
+		return
+	}
+	if tr.SolanaChainID == "" {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("missing solanaChainID"))
+		return
+	}
+	chain, err := solanaChains.Chain(c.Request.Context(), tr.SolanaChainID)
+	switch err {
+	case solana.ErrChainIDInvalid, solana.ErrChainIDEmpty:
+		jsonAPIError(c, http.StatusBadRequest, err)
+		return
+	case nil:
+		break
+	default:
+		jsonAPIError(c, http.StatusInternalServerError, err)
+		return
+	}
+
+	if tr.From.IsZero() {
+		jsonAPIError(c, http.StatusUnprocessableEntity, errors.Errorf("source address is missing: %v", tr.From))
+		return
+	}
+
+	if tr.Amount == 0 {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("amount must be greater than zero"))
+		return
+	}
+
+	fromKey, err := tc.App.GetKeyStore().Solana().Get(tr.From.String())
+	if err != nil {
+		jsonAPIError(c, http.StatusUnprocessableEntity, errors.Errorf("fail to get key: %v", err))
+		return
+	}
+
+	txm := chain.TxManager()
+	var reader client.Reader
+	reader, err = chain.Reader("")
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, errors.Errorf("chain unreachable: %v", err))
+		return
+	}
+
+	if !tr.AllowHigherAmounts {
+		err = solanaValidateBalance(reader, tr.From, tr.Amount)
+		if err != nil {
+			jsonAPIError(c, http.StatusUnprocessableEntity, errors.Errorf("failed to validate balance: %v", err))
+			return
+		}
+	}
+
+	blockhash, err := reader.LatestBlockhash()
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, errors.Errorf("failed to get latest block hash: %v", err))
+		return
+	}
+
+	tx, err := solanaGo.NewTransaction(
+		[]solanaGo.Instruction{
+			system.NewTransferInstruction(
+				tr.Amount,
+				tr.From,
+				tr.To,
+			).Build(),
+		},
+		blockhash.Value.Blockhash,
+		solanaGo.TransactionPayer(tr.From),
+	)
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, errors.Errorf("failed to create tx: %v", err))
+		return
+	}
+
+	_, err = tx.Sign(
+		func(key solanaGo.PublicKey) *solanaGo.PrivateKey {
+			if tr.From.Equals(key) {
+				r := solanaGo.PrivateKey(fromKey.Raw())
+				return &r
+			}
+			return nil
+		},
+	)
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, errors.Errorf("failed to sign tx: %v", err))
+		return
+	}
+
+	err = txm.Enqueue("", tx)
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, errors.Errorf("transaction failed: %v", err))
+		return
+	}
+
+	jsonAPIResponse(c, map[string]interface{}{"message": "transaction sent"}, "solana_tx")
```

Comment:
The current API doesn't return anything, but we could return their input plus block hash? 
- [ ] Improve `txs create` response

Reply:
didn't add the blockhash, but added the inputs: https://github.com/smartcontractkit/chainlink/pull/6231/commits/af3d35a777ab101598bc598c6aa72bdadf619be7